tests/FlatBuffers.Test/FlatBuffersExampleTests.cs

```diff
@@ -287,6 +287,62 @@ public void CanReadCppGeneratedWireFile()
             TestObjectAPI(Monster.GetRootAsMonster(bb));
         }
 
+#if ENABLE_JSON_SERIALIZATION
+        [FlatBuffersTestMethod]
+        public void CanReadJsonFile()
+        {
+            var jsonText = File.ReadAllText(@"Resources/monsterdata_test.json");
+            var mon = MonsterT.DeserializeFromJson(jsonText);
+
+            Assert.AreEqual(1.0f, mon.Pos.X);
```

Comment:
Can these tests be shared with the existing test code much like the C++ tests do?

Reply:
I shared the existing `TestBuffer`  method.